### PR TITLE
rebase: update node driver registar to latest version

### PR DESCRIPTION
**Pull request by humblec**
v2.6.0 is the latest version of the same and this commit update it.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: charts/ceph-csi-cephfs/values.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ nodeplugin:
   registrar:
     image:
       repository: registry.k8s.io/sig-storage/csi-node-driver-registrar
-      tag: v2.5.1
+      tag: v2.6.0
```
Comment (Madhu-1) [inline charts/ceph-csi-cephfs/values.yaml:90]: update it in the helm documentation also

File: charts/ceph-csi-rbd/values.yaml [hunk 2]
```diff
@@ -107,7 +107,7 @@ nodeplugin:
   registrar:
     image:
       repository: registry.k8s.io/sig-storage/csi-node-driver-registrar
-      tag: v2.5.1
+      tag: v2.6.0
```
Comment (Madhu-1) [inline charts/ceph-csi-rbd/values.yaml:113]: update it in the helm documentation also

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Comment by humblec** [comment]
@Mergifyio refresh

**Comment by humblec** [comment]
@Mergifyio rebase

**Comment by humblec** [comment]
@Mergifyio requeue

**Pull request merged**